testutil/integration/model_test.go

```diff
@@ -18,73 +28,127 @@ import (
 var _ = Describe("Model Registry", func() {
 	const timeout = time.Second * 5
 	const interval = time.Second * 1
+	type Format string
+
+	const (
+		FormatSavedModel  Format = "SavedModel"
```

Comment:
You can refer to it directly here, right? There is no need to repeat the definition

https://github.com/kleveross/klever-model-registry/blob/a480cc39ecd041b164383e35799785e3433461a9/pkg/apis/modeljob/v1alpha1/modeljob_types.go#L124

Reply:
OK, will be changed